src/storage/engine/mod.rs

```diff
@@ -231,6 +233,8 @@ mod tests {
                 .unwrap();
             must_put_cf(e.as_ref(), "cf", b"k", b"v1");
         }
+        // Sleep a while to make sure the `LOCK` file is deleted.
+        thread::sleep(Duration::from_millis(100));
```

Comment:
I also thought it's okay to reopen it as soon as the old one is dropped. But error message shows that the LOCK file is not deleted instantly somehow.